vdr/store/bbolt.go

```diff
@@ -16,62 +16,56 @@
 package store
 
 import (
+	"context"
 	"encoding/json"
 	"fmt"
-	"os"
-	"path"
 
 	"github.com/nuts-foundation/go-did/did"
+	"github.com/nuts-foundation/go-stoabs"
 	"github.com/nuts-foundation/nuts-node/core"
 	"github.com/nuts-foundation/nuts-node/crypto/hash"
-	"go.etcd.io/bbolt"
-
+	"github.com/nuts-foundation/nuts-node/storage"
 	vdr "github.com/nuts-foundation/nuts-node/vdr/types"
 )
 
 const (
-	// latestBucket has DID as key and latest metadata reference as value
-	latestBucket = "latest"
-	// metadataBucket has the metadata reference (DID concatenated with version number, starting at 0) as key and the metadataRecord as value
-	metadataBucket = "metadata"
-	// transactionIndexBucket has the transaction reference as key and metadata reference as value
-	transactionIndexBucket = "txRef"
-	// documentsBucket has payload hash as key and document as value
-	documentBucket = "documents"
+	// latestShelf has DID as key and latest metadata reference as value
+	latestShelf = "latest"
+	// metadataShelf has the metadata reference (DID concatenated with version number, starting at 0) as key and the metadataRecord as value
+	metadataShelf = "metadata"
+	// transactionIndexShelf has the transaction reference as key and metadata reference as value
+	transactionIndexShelf = "txRef"
+	// documentShelf has payload hash as key and document as value
+	documentShelf = "documents"
 )
 
-type bboltStore struct {
-	db *bbolt.DB
+type store struct {
```

Comment:
filename may be changed as well.

Reply:
I will change all files in a different PR. Github picks this up as a delete old + add new for some files instead of a rename. This is hard to review.